### PR TITLE
Add net area column to envelope summary

### PR DESCRIPTION
**Pull request by JasonGlazer**
Addresses #4177
Added new "Net Area" column to the Envelope Summary report Opaque Exterior table that represents the gross wall area minus the window and door areas.

### CODE REVIEW

**Comment by JasonGlazer** [comment]
The failed tests seem to be due to #5393 but will merge with the latest develop and run the tests again to make sure.

File: tst/EnergyPlus/unit/OutputReportTabular.unit.cc [hunk 1]
```diff
@@ -311,6 +311,20 @@ TEST( OutputReportTabularTest, GetColumnUsingTabs )
...
+TEST( OutputReportTabularTest, ConfirmConvertToEscaped )
+{
+	ShowMessage( "Begin Test: OutputReportTabularTest, ConfirmConvertToEscaped" );
+
+	EXPECT_EQ( "String with &gt; in it", ConvertToEscaped( "String with > in it" ) );
+	EXPECT_EQ( "String with &lt; in it", ConvertToEscaped( "String with < in it" ) );
+	EXPECT_EQ( "String with &amp; in it", ConvertToEscaped( "String with & in it" ) );
+	EXPECT_EQ( "String with &quot; in it", ConvertToEscaped( "String with \" in it" ) );
+	EXPECT_EQ( "String with &apos; in it", ConvertToEscaped( "String with ' in it" ) );
+//	EXPECT_EQ( "String with * in it", ConvertToEscaped( "String with � in it" ) );    //degree symbol - does not work since C++ wants to substitute \ code
```
Comment (JasonGlazer) [inline tst/EnergyPlus/unit/OutputReportTabular.unit.cc]: @Myoldmopar @mbadams5 I could use some advice on how to detect degree symbol ° (a.k.a \xB0). I tried a few things and also tried modifying ConvertToEscaped but could not figure it out.

Comment (mbadams5) [inline tst/EnergyPlus/unit/OutputReportTabular.unit.cc]: I'll see what I can figure out, but I do have a question. Why use `*` instead of `&deg;`?

Comment (JasonGlazer) [inline tst/EnergyPlus/unit/OutputReportTabular.unit.cc]: If I remember correctly, the convert routine is being used when generating XML files and &deg; did not seem to be supported correctly with the XML parser I was testing with so \* seemed as close as I could get.

Comment (mbadams5) [inline tst/EnergyPlus/unit/OutputReportTabular.unit.cc]: Well I guess what i'm asking is do you want to use `*` or use `&deg;`, which is the correct HTML escape for degree? It is more inline with the other escapes.

Comment (JasonGlazer) [inline tst/EnergyPlus/unit/OutputReportTabular.unit.cc]: We probably should use &deg;

Comment (mbadams5) [inline tst/EnergyPlus/unit/OutputReportTabular.unit.cc]: I am confused. You need to escape °, correct? So what do you want the escaped value to be? `*` or `&deg;`?

Comment (JasonGlazer) [inline tst/EnergyPlus/unit/OutputReportTabular.unit.cc]: Sorry, forgot to use backquotes. I meant: `&deg;`

**Comment by mbadams5** [comment]
@JasonGlazer I just refactored the function to handle unicode better. It is still a little kludgy but it is better. Look over my changes and the added unit tests. It should handle degree symbol now.

**Comment by JasonGlazer** [comment]
Thanks @mbadams5. That was a lot of refactoring. It looks like it will handle a lot more cases!

**Comment by nrel-bot-2** [comment]
@JasonGlazer @lgentile it has been 14 days since this pull request was last updated.

**Comment by mjwitte** [comment]
@JasonGlazer The output change should be added to [OutputChanges8-4-0-to-8-5-0.md](https://github.com/NREL/EnergyPlus/blob/AddNetAreaColumnToEnvelopeSummary/src/Transition/OutputRulesFiles/OutputChanges8-4-0-to-8-5-0.md)
Also, please merge up to current develop.  Code itself looks good.

**Comment by mjwitte** [comment]
Tested with several files (with and without zone multipliers and zone group multipliers).  All the net areas check out OK.  Note that CI doesn't report the tablediffs because the tables changed size so they get skipped.

**Pull request merged**